### ISSUE TITLE
Not Sideloading

### ISSUE THREAD

**Issue by LavaLad** [pwned4ever/Th0r_Freya#5]
Whenever I try to sideload Freya it shows that it has a virus or some potentially unwanted software

**Comment by Voxeliorn** [comment]
thats a false positive you can ignore that

**Comment by LavaLad** [comment]
I figured how to bypass that but then when I try to jailbreak it shows as failed to remount or smt

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pwned4ever** [comment]
Check your settings iPhone storage for the system update file. Delete it. If you're on the latest version. I'd suggest trying to restore the device or erase all content and settings and try again. That's if the update file error was the prompt that appeared or if it said failed to bootstrap so it's gonna remount and start over

Sent from my iPhone

On Oct 25, 2022, at 5:24 AM, LavaLad ***@***.***> wrote:

﻿

I figured how to bypass that but then when I try to jailbreak it shows as failed to remount or smt

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by LavaLad** [comment]
I did that too and it worked. But then when I try to jailbreak it just restarts and didn't even show any problems and when I tried it a couple of times it still doesn't work (am using sideloadly btw)
Sorry for the trouble and problems

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pwned4ever** [comment]
What version of freya are you trying? When it doubt always use the latest, which now is 1.2.5

Sent from my iPhone

On Oct 25, 2022, at 3:04 PM, LavaLad ***@***.***> wrote:

﻿

I did that too and it worked. But then when I try to jailbreak it just restarts and didn't even show any problems and when I tried it a couple of times it still doesn't work (am using sideloadly btw)
Sorry for the trouble and problems

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by LavaLad** [comment]
I tried the 1.2.5 version but nothing works....

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pwned4ever** [comment]
Elaborate on nothing works! That's basically saying the whole project is worthless. If you can't get it working I'd say, use chimera or Unc0ver then. Best of luck with those other jailbreaks

Sent from my iPhone

On Oct 26, 2022, at 1:55 AM, LavaLad ***@***.***> wrote:

﻿

I tried the 1.2.5 version but nothing works....

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by LavaLad** [comment]
Whoa chill bruh! I never said it's worthless. It works on other phones but didn't seem to be working on mine. I didn't mean to offend you though. It first shows failed to remount, and as you said before I'm supposed to go in the iPhone's storage and delete the system file but I found no such thing there. Maybe a picture explaining it would be better


________________________________

**Comment by pwned4ever** [comment]
Appreciate your reply. A video might help to verify issues. However the more info I receive from you the better I can assist. 
Like what device exactly are you using freya on, what is the version of iOS your using freya on? What version of freya have you tried, have you previously used another Jailbreak before attempting to use freya?
If the device is updated to 12.5.6, there is no update file, however I've had one user, state they get that error on iOS 12.5.6, which prompts about the update file. If your error is different then, steps what to do next would be different. Other than removing the jailbreak. Wiping the device clean amd starting from a stock fresh device. With 0 remanence of any Jailbreaks left on device. 
Show me as much video as possible please. If you have Twitter, share it with me on there please. @shogunpwnd

**Comment by pwned4ever** [comment]
Please tell me, did you do an Ota update or did you use a computer and a ipsw? Have you tried to erase all content and settings and start from a freshly stock device? Without reatoring anything to the ipad before using the jailbreak.

**Comment by LavaLad** [comment]
I honestly didn't understand what you meant. Could you please dumb it down for me. I'm using an iPhone 6 plus. I haven't tried with a freshly stock device yet


________________________________

**Comment by pwned4ever** [comment]
What I'm asking is, did you update the iPhone 6 to 12.5.6 using the settings app on the device itself? Or did you use a computer and download the update file onto a computer and install it via a computer? Also. What did you use previously on the device before freya? What version of software was it on before you updated to iOS 12.5.6? Or did you buy the device with that software on it?

Sent from my iPhone

On Oct 30, 2022, at 12:12 PM, LavaLad ***@***.***> wrote:

﻿

I honestly didn't understand what you meant. Could you please dumb it down for me. I'm using an iPhone 6 plus. I haven't tried with a freshly stock device yet


________________________________

**Comment by LavaLad** [comment]
Thank you! Makes it much simpler. I did the update using the settings app and I once tried using Chimera on 12.5.5 but that didn't seem to work.


________________________________

**Comment by pwned4ever** [comment]
Ok, so I need you to try these steps. Considering you have a computer at home to do this with?
If you don't have a computer. Then it's gonna be hard to erase the device and backup everything in the iCloud.
You'll need to erase the device and start over with a freshly cleaned stock version of ios 12.5.6.
Either by downloading the 12.5.6 ipsw and loading it on to the device via a computer/Mac or you must erase all content and settings. Using the settings app on the device itself. To clean and freshly install a stock ios 12.5.6 system on your device. After you've erased the device. Do not use a backup of your data onto the device yet. Use the jailbreak before you restore your data onto the device.

Sent from my iPhone

On Oct 31, 2022, at 7:12 AM, LavaLad ***@***.***> wrote:

﻿

Thank you! Makes it much simpler. I did the update using the settings app and I once tried using Chimera on 12.5.5 but that didn't seem to work.


________________________________

**Comment by LavaLad** [comment]
I honestly don't trust iCloud that much so I don't have that much hope that it'll work. But thanks I'll try and let you know


________________________________
